module/signature/aggregation.go

```diff
@@ -209,7 +206,7 @@ func (s *SignatureAggregatorSameMessage) Aggregate() ([]int, crypto.Signature, e
 	if !ok {
 		// check for identity signature (invalid aggregated signature)
 		if crypto.IsBLSSignatureIdentity(aggregatedSignature) {
-			return nil, nil, InvalidAggregatedSignatureError
+			return nil, nil, fmt.Errorf("invalid aggregated signature: %w", ErrIdentitySignature)
```

Comment:
nit and can be ignored: I think the outer and internal error describe exactly the same idea, maybe we could return `nil, nil, ErrIdentitySignature` in this case?

Reply:
Well, that depends on the perspective. I assumed that `ErrIdentitySignature` might be used in other places too, and not only here. In context, your proposed error description ties the error exactly to this use case. See [my comment above](https://github.com/onflow/flow-go/pull/3677#discussion_r1051313145) for details